test/utils/createApp.ts

```diff
@@ -0,0 +1,15 @@
+import { Page } from '@playwright/test';
+
+export default async function createApp(page: Page, name: string, dom?: string) {
+  await page.locator('button:has-text("create new")').click();
+
+  await page.fill('[role="dialog"] label:has-text("name")', name);
```

Comment:
We could improve dialog selectors a bit: https://github.com/mui/mui-toolpad/pull/838/files#diff-e537b99227068da9665d973e6573a1cd6da8d36be778d2aba43b159890a43d99R30

But we can merge this as is and use the ToolpadHome model as an optimization later

Reply:
sure, i just took this function from the other tests where it was
we can use that model if we merge it first